PackageNamespacesExample/nuget.config

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>

```

Comment:
add `<add key="globalPackagesFolder" value="c:\packages" />` to the `config` as per https://github.com/NuGet/nuget.github.com/pull/214/files#diff-75614b5ad7a9e3e38a3520a89ee7c69a63a052fb09c82a7c91bef7004e29d7c6R51 annoucement.

Reply:
I don't think examples should redirect to directories *outside* of their own directory structure.

Reply:
Fixed, it redirects to a local folder right now.